### ISSUE TITLE
How to refer to the monaco type

### ISSUE THREAD

**Issue by DreamGhostStar** [suren-atoyan/monaco-react#495]
**Describe the bug**
**@monaco-editor/react** seems to request **monaco-editor** files by way of cdn, so how do I use the `monaco.editor.IStandaloneCodeEditor` type in my code, I get an error when I refer to monaco-editor/esm/vs/editor/editor.api
![image](https://github.com/suren-atoyan/monaco-react/assets/48384290/d415f0b7-e976-4ec5-8cfc-c49bc504ec58)
It seems impossible to introduce, but I have to use this type in `useRef`.

**version**
**@monaco-editor/react**: 4.4.6

**Comment by suren-atoyan** [comment]
@DreamGhostStar try

```js
import { type editor } from 'monaco-editor';

...

editor.IStandaloneCodeEditor

```

**Comment by DreamGhostStar** [comment]
@suren-atoyan No, it doesn't seem to work because I don't have a monaco-editor dependency
![image](https://github.com/suren-atoyan/monaco-react/assets/48384290/b558010b-f9f6-46fc-a68e-ca2b8c112d5c)
